modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -0,0 +1,500 @@
+= snowflake_streaming
+// tag::single-source[]
+:type: output
+:page-beta: true
+:categories: ["Services"]
```

Comment:
As there are now 2 snowflake outputs, how do users know which one to use? Are they for different use cases?

Reply:
People should migrate from `snowflake_put` to `snowflake_streaming` it's faster, cheaper and easier to use

Reply:
I've added a note to both connectors. See what you think.

Reply:
Do we want to deprecate `snowflake_put`? Will we still maintain both if one is better and recommended?

Reply:
For the moment, Ash has said they will both be kept until the `snowflake_streaming` output in proven.